CHANGELOG.md

```diff
@@ -1,11 +1,16 @@
 # Ember Data Changelog
 
-## Release 3.24.0-beta.0 (November 30, 2020)
-- [#7350](https://github.com/emberjs/data/pull/7350) [Chore]: convert adapters to class syntax (#7350)
-- [#7359](https://github.com/emberjs/data/pull/7359) [CI]: bump failure asset size to 75 bytes (#7359)
+## Release 3.24.0 (January 4, 2021)
+- [#7350](https://github.com/emberjs/data/pull/7350) [Chore]: convert adapters to class syntax
+- [#7359](https://github.com/emberjs/data/pull/7359) [CI]: bump failure asset size to 75 bytes
 - [#7369](https://github.com/emberjs/data/pull/7369) [CHORE] Bump @ember/ordered-set
 - [#7370](https://github.com/emberjs/data/pull/7370) Don't expose the value of a model's attribute in assertions (#7370)
-- [#7363](https://github.com/emberjs/data/pull/7363) [BUGFIX identifiers] Address issue with polymorphic findRecord  (#7363)
+- [#7363](https://github.com/emberjs/data/pull/7363) [BUGFIX identifiers] Address issue with polymorphic findRecord
+- [#7262](https://github.com/emberjs/data/pull/7262) [CHORE]: Extract internalModel access to identifiers for Relationships
+
+## Release 3.23.0 (November 30, 2020)
+
+## Release 3.22.0 (November 5, 2020)
```

Comment:
I was having trouble figuring out what should go in these slots

Reply:
The release channel, should have them we should backport

Reply:
Good call https://github.com/emberjs/data/pull/7383/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR3

Reply:
https://github.com/emberjs/data/pull/7383/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR10
You can copy paste